nistats/datasets.py

```diff
@@ -334,69 +485,134 @@ def fetch_spm_auditory(data_dir=None, data_name='spm_auditory',
-
-        """
-
-        if not os.path.exists(subject_dir):
+    if not os.path.exists(subject_dir):
+        _download_spm_auditory_data(data_dir, subject_dir, subject_id)
+    spm_auditory_data = _prepare_downloaded_spm_auditory_data(subject_dir)
+    try:
+        spm_auditory_data['paradigm']
+    except KeyError:
+        events_filepath = _make_path_events_file_spm_auditory_data(
+                                                            spm_auditory_data)
+        if not os.path.isfile(events_filepath):
+            _make_events_file_spm_auditory_data(events_filepath)
+        spm_auditory_data['paradigm'] = events_filepath
+    return spm_auditory_data
+
+
+def _get_func_data_spm_multimodal(subject_dir, session, _subject_data):
+    session_func = sorted(glob.glob(
+            os.path.join(
+                    subject_dir,
+                    ("fMRI/Session%i/fMETHODS-000%i-*-01.img" % (
+                        session + 1, session + 5)))))
+    if len(session_func) < 390:
+        print("Missing %i functional scans for session %i." % (
+            390 - len(session_func), session))
+        return None
+    
+    _subject_data['func%i' % (session + 1)] = session_func
+    return _subject_data
+
+
+def _get_session_trials_spm_multimodal(subject_dir, session, _subject_data):
+    sess_trials = os.path.join(
+            subject_dir,
+            "fMRI/trials_ses%i.mat" % (session + 1))
+    if not os.path.isfile(sess_trials):
+        print("Missing session file: %s" % sess_trials)
+        return None
+    
+    _subject_data['trials_ses%i' % (session + 1)] = sess_trials
+    return _subject_data
+
+
+def _get_anatomical_data_spm_multimodal(subject_dir, _subject_data):
+    anat = os.path.join(subject_dir, "sMRI/smri.img")
+    if not os.path.isfile(anat):
+        print("Missing structural image.")
+        return None
+    
+    _subject_data["anat"] = anat
+    return _subject_data
+
+
+def _glob_spm_multimodal_fmri_data(subject_dir):
+    """glob data from subject_dir."""
+    _subject_data = {'slice_order': 'descending'}
+
+    for session in range(2):
+        # glob func data for session s + 1
+        _subject_data = _get_func_data_spm_multimodal(subject_dir, session, _subject_data)
+        if not _subject_data:
             return None
+        # glob trials .mat file
+        _subject_data = _get_session_trials_spm_multimodal(subject_dir, session, _subject_data)
+        if not _subject_data:
+            return None
+        try:
+            paradigm = _make_events_file_spm_multimodal_fmri(_subject_data, session)
+        except MatReadError as mat_err:
+            warnings.warn('{}. An events.tsv file cannot be generated'.format(str(mat_err)))
+        else:
+            events_filepath = _make_events_filepath_spm_multimodal_fmri(_subject_data, session)
+            paradigm.to_csv(events_filepath, sep='\t', index=False)
+            _subject_data['events{}'.format(session+1)] = events_filepath
```

Comment:
pep8 'session + 1'

Reply:
In standalone expressions, not in function/method args.

Reply:
Regardless, I'll obviate this concern soon